platforms/platform-alexa/src/output/templates/AudioPlayerPlayOutput.ts

```diff
@@ -0,0 +1,56 @@
+import { BaseOutput, Output, OutputOptions, OutputTemplate } from '@jovotech/framework';
+import { AudioItem, PlayBehavior, PlayBehaviorLike } from '../models';
+import _merge from 'lodash.merge';
+
+export interface AudioPlayerPlayOutputOptions extends OutputOptions {
+  playBehavior?: PlayBehaviorLike;
+  audioItem?: AudioItem;
+}
+
+@Output()
+export class AudioPlayerPlayOutput extends BaseOutput<AudioPlayerPlayOutputOptions> {
+  getDefaultOptions(): AudioPlayerPlayOutputOptions {
+    return {
+      playBehavior: PlayBehavior.ReplaceAll,
+      audioItem: {
+        stream: {
+          url: '',
+          token: '',
+          offsetInMilliseconds: 0,
+        },
+      },
+    };
+  }
+
+  build(): OutputTemplate | OutputTemplate[] {
+    if (this.options.audioItem) {
+      this.options.audioItem.stream.offsetInMilliseconds =
+        this.options.audioItem.stream.offsetInMilliseconds || 0;
+      this.options.audioItem.stream.token =
```

Comment:
could we maybe outsource this step into a helper method for readability? so that people see at first glance what's happening here. or add a comment?

Reply:
Make sense, thx